src/common/Collision/Management/VMapManager2.cpp

```diff
@@ -279,7 +280,7 @@ namespace VMAP
         return false;
     }
 
-    WorldModel* VMapManager2::acquireModelInstance(const std::string& basepath, const std::string& filename)
+    WorldModel* VMapManager2::acquireModelInstance(const std::string& basepath, const std::string& filename, uint32 flags/* Only used when creating the model */)
```

Comment:
I would rather keep this as it is now and make a setter for flags. Just personal preferences because it feels wrong imo